### PR TITLE
community/bitcoin: upgrade to 0.19.0

### PR DESCRIPTION
**Pull request by itoffshore**
* https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.19.0.md

### CODE REVIEW

**Comment by Cogitri** [comment]
Can you look into the failure on Drone CI?

**Comment by itoffshore** [comment]
@Cogitri - `drone` is complaining with:

`error: sha1 information is lacking or useless (community/bitcoin/APKBUILD)`

 but we use `sha512` checksums in `APKBUILD` so I don't think this is an error I can fix ?

**Comment by Cogitri** [comment]
Oh right, I think it fails to clone the commit, read that wrong, sorry!

File: community/bitcoin/APKBUILD [hunk 1]
```diff
@@ -1,21 +1,21 @@
 # Contributor: Natanael Copa <ncopa@alpinelinux.org>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=bitcoin
-pkgver=0.18.1
+pkgver=0.19.0
 _ver=${pkgver/_/}
 pkgrel=3
```
Comment (Cogitri) [inline community/bitcoin/APKBUILD]: Please reset the pkgrel to 0

**Comment by Cogitri** [comment]
Ah?

**Comment by itoffshore** [comment]
I only deleted my local branch & force pushed the update to resolve the merge conflict

**Comment by Cogitri** [comment]
Ah, but now this PR doesn't contain any commits

**Pull request closed without merging**